lib/github_service/commands/cross_repo_test.rb

```diff
@@ -277,7 +278,7 @@ def self.fetch_manageiq_release_repo_groups
 
         uri = URI.parse("https://raw.githubusercontent.com/ManageIQ/manageiq-release/master/config/repos.sets.yml")
         response = Net::HTTP.get_response(uri)
-        return {} if response.code != 200
+        return {} if response.value
```

Comment:
I think this is wrong.  From what I read `value` will raise an exception if the error code is not 200.  But this code will return {} if it doesn't fail?

Reply:
:facepalm: you're right updated to call `response.value` and rescue the exception and return `{}`